src/render.rs

```diff
@@ -141,11 +142,31 @@ impl<'a> Default for MarkdownRenderer<'a> {
 /// let text = "[Rust](https://rust-lang.org/) is an awesome *systems programming* language!";
 /// let rendered = markdown_to_html(text)?;
 /// ```
-pub fn markdown_to_html(text: &str) -> CargoResult<String> {
+fn markdown_to_html(text: &str) -> CargoResult<String> {
```

Comment:
If we drop the `pub` modifier on `markdown_to_html`, we probably want to drop it for `struct MarkdownRenderer` too ?

Reply:
Yes!